src/services/netflix/NetflixApi.ts

```diff
@@ -188,7 +188,7 @@ class _NetflixApi extends ServiceApi {
 				});
 				this.session = this.extractSession(responseText);
 			}
-			if (this.session) {
+			if (this.session && this.session.profileName) {
```

Comment:
Why not `this.session?.profileName`?

Reply:
Sure, that optimizes the code a bit, I just used a similar style as in the [`checkLogin()`](https://github.com/trakt-tools/universal-trakt-scrobbler/blob/14efdd20bccc0815f5cc5cad5063cd2b16145024/src/services/netflix/NetflixApi.ts#L206)-function. But I'll go ahead and update it.